### PR TITLE
Add an api for giving local-dynamo more heap

### PR DESCRIPTION
**Pull request by nicks**
Hello @kylehg, @chaosgame, 

Please review the following commits I made in branch 'nick-api'.

61beeba378422b5d5afadd56569324ce541a749c (2015-06-22 15:04:14 -0700)
Add an api for giving local-dynamo more heap

R=@kylehg
R=@chaosgame

Test plan: 'not tested'

### CODE REVIEW

**Comment by kylehg** [comment]
Oh nice, I didn't know you could do this — I suspect this could've saved us a good bit of trouble before

File: lib/launch.js [hunk 1]
```diff
@@ -8,11 +8,33 @@ var cp = require('child_process')
 var path = require('path')
 
 /**
- * @param {?string} databaseDir The location of database files. Will run in memory if null.
- * @param {number} port
+ * An options object:
+ *   port: {number} - The port to run on. Required.
+ *   dir: {?string=} - The location of database files. Optional Will run in-memory if null.
+ *   heap: {?string=} - The amount of heap space, e.g., 512m. If not specified, uses JVM defaults.
```
Comment (kylehg) [inline lib/launch.js]: Maybe link to something indicating more info about the format of this option, e.g., http://stackoverflow.com/questions/5374455/what-does-java-option-xmx-stand-for

**Comment by kylehg** [comment]
LGTM!

**Comment by nicks** [comment]
more docs added!

**Pull request merged**